examples/kafka-service-error-handling/kafka_service_error_handling.md

```diff
@@ -0,0 +1,18 @@
+# Kafka service - Error handling
+
+The `kafka:Service` has an `onError` method that is called when an internal error occurs before the `onConsumerRecord` method is called. These errors could include payload binding errors or constraint validation errors. The `onError` method allows the user to handle the error as needed. If an error is returned from the `onConsumerRecord` method, it is logged to the console, and the next polling cycle continues.
```

Comment:
The kafka:Service has an onError method which is invoked when `kafka:Listener` triggers errors  before the onConsumerRecord method is invoked. These errors could include payload binding errors or constraint validation errors. The onError method allows the user to handle these errors as needed. If there is no `onError` these are logged to the console with the stack-trace. In addition, if an error is returned from the onConsumerRecord method, it is logged to the console and the next polling cycle continues.

Reply:
Updated